lib/p2p/NodeList.ts

```diff
@@ -141,7 +141,9 @@ class NodeList extends EventEmitter {
         promises.push(this.setBanStatus(node, true));
         this.bannedNodes.add(node.nodePubKey);
         this.nodes.delete(node.nodePubKey);
-        this.emit('node.ban', nodePubKey);
+
+        const events = await this.repository.getReputationEvents(node);
```

Comment:
Sending all reputation events ever seems excessive, I think it would be better to only retrieve something like the last 5 or 10 negative events (no point in sending positive ones). If you don't want to handle that in this PR you can just put a TODO, but it shouldn't be too hard to query a subset of the events.

Reply:
This is marked as resolved, but I can't see @sangaman's comment being addressed. Sorry if I'm missing something.

Reply:
Yes I'm unresolving as it seems like it was overlooked. It's not critical or anything but a TODO would be good so that it's not forgotten.

Reply:
See my comment on the main thread.

Reply:
Oops my mistake, thank you!